ruby/lang/weak_encryption.yml

```diff
@@ -1,8 +1,8 @@
 patterns:
   - pattern: |
-      Digest::SHA1.hexidigest()
+      Digest::SHA1.hexdigest()
```

Comment:
I think you are still missing the OpenSSL version https://ruby-doc.org/stdlib-2.5.1/libdoc/openssl/rdoc/OpenSSL/Digest.html